tools/cjxl_ng_main.cc

```diff
@@ -149,49 +150,34 @@ DEFINE_int32(
 
 DEFINE_int64(
     center_x, -1,
-    // TODO(tfish): Clarify if this is really the comment we want here.
     "Determines the horizontal position of center for the center-first "
     "group order. The value -1 means 'use the middle of the image', "
-    // TODO(tfish): Clarify if encode.h has an off-by-one in the
-    // upper limit here.
-    "other values 0..(xsize-1) set this to a particular coordinate.");
+    "other values [0..xsize) set this to a particular coordinate.");
 
 DEFINE_int64(center_y, -1,
-             // TODO(tfish): Clarify if this is really the comment we want here.
              "Determines the vertical position of center for the center-first "
              "group order. The value -1 means 'use the middle of the image', "
-             // TODO(tfish): Clarify if encode.h has an off-by-one in the
-             // upper limit here.
-             "other values 0..(ysize-1) set this to a particular coordinate.");
+             "other values [0..ysize) set this to a particular coordinate.");
 
 DEFINE_int64(num_threads, 0,
-             // TODO(tfish): Sync with team about changed meaning of 0 -
-             // was: No multithreaded workers. Is: use default number.
-             "Number of worker threads (0 == use machine default).");
+             "Number of worker threads (0 == use machine default, "
+             "-1 == do not use multithreading).");
```

Comment:
To me it makes more sense to have -1 = default, 0 = everything in main thread. We use -1 as the default in just about every other setting, so why not here?

Reply:
Switched -1 and 0.